function-images/tests/chained-function-serving/consumer/consumer.go

```diff
@@ -60,17 +63,21 @@ func (s *consumerServer) ConsumeStream(stream pb.ProducerConsumer_ConsumeStreamS
 
 func main() {
 	port := flag.Int("ps", 80, "Port")
+	url := flag.String("zipkin", "http://zipkin.istio-system.svc.cluster.local:9411/api/v2/spans", "zipkin url")
 	flag.Parse()
 
 	log.SetOutput(os.Stdout)
 
+	shutdown := tracing.InitBasicTracer(*url, "consumer")
```

Comment:
we need to distinguish the QP from the function instance. please add "*-fu" suffix as in "cons-fu" to specify that this is not a part of the cloud infra. We need to specify that Zipkin can visualize only 7 characters in a graph
![image](https://user-images.githubusercontent.com/15651770/122747485-04eb2480-d283-11eb-8337-ece1c830cf19.png)